test/e2e/handler/utils.go

```diff
@@ -612,9 +618,14 @@ func routeDest(node, destIP string) string {
 	return gjson.ParseBytes(currentStateJSON(node)).Get(path).String()
 }
 
-func routeNextHopInterface(node, destIP string) AsyncAssertion {
+// routeNextHopInterface checks if a route with destIP exists in table tableID. If tableID is the empty string, use
+// the default table-id (254).
+func routeNextHopInterface(node, destIP, tableID string) AsyncAssertion {
```

Comment:
Same here let's keep `routeNextHopInterface` and create new `routeNextHopInterfaceWithTableID`

Reply:
Yep looks better, done